controller/handler/badge.go

```diff
@@ -199,7 +206,7 @@ func getBadgeColorFromUptime(uptime float64) string {
 	return badgeColorHexVeryBad
 }
 
-func generateResponseTimeBadgeSVG(duration string, averageResponseTime int) []byte {
+func generateResponseTimeBadgeSVG(duration string, averageResponseTime int, key string, config *config.Config) []byte {
```

Comment:
Can you rename `config *config.Config` to `cfg *config.Config` to avoid package name collision?